File: easy/kotlin/3110-score-of-a-string/main.kt
package easy.kotlin.`3110-score-of-a-string`

import kotlin.math.abs

class Solution {
    fun scoreOfString(s: String): Int {
        var sum = 0;

        for (i in 1 until s.length)
            sum += abs(s[i] - s[i-1])

        return sum;
    }
}

//fun main(){
//    println(Solution().scoreOfString("hello"))
//}

File: easy/c/3110-score-of-a-string/main.c
#include <stdio.h>
#include <stdlib.h>

int scoreOfString(char* s) {
    int sum = 0;

    for(int i = 0; s[i + 1] != '\0'; i++)
        sum += abs(s[i] - s[i + 1]);

    return sum;
}

// int main() {
//     printf("%i", scoreOfString("hello"));
//
//     return 0;
// }